### PR TITLE
Dockerise app for development/testing

### PR DESCRIPTION
**Pull request by archferns**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/CACP-337)
Dockerise app to run both dev and specs using compose.

## Checklist

Before you ask people to review this PR:

- [ ] Tests and linters should be passing
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: helm_deploy/laa-court-data-adaptor/templates/_environment.tpl [hunk 1]
```diff
@@ -14,6 +14,8 @@ env:
       secretKeyRef:
         name: ec-cluster-output
         key: url
+  - name: RAILS_ENV
+    value: 'production'
```
Comment (richrace) [inline helm_deploy/laa-court-data-adaptor/templates/_environment.tpl:18]: Should this be hardcoded as `production`?

**Comment by richrace** [comment]
Once this goes into master, we'll need to update any related Readmes for the new instructions to get this running locally. 

E.g. https://github.com/ministryofjustice/laa-court-data-ui/blob/master/docs/development.md

File: helm_deploy/laa-court-data-adaptor/templates/_environment.tpl (hunk 1 above)
Comment (archferns) [reply helm_deploy/laa-court-data-adaptor/templates/_environment.tpl:18]: The idea was to run the cloud version in `production`, and control via related ENV vars, like `SENTRY_ENV` and so on. Happy to change to a different approach though.

Comment (archferns) [reply helm_deploy/laa-court-data-adaptor/templates/_environment.tpl:18]: Seeing a similar approach in [apply](https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/9087b689771dcfe37db7ee4891be037a8ce5977a/helm_deploy/apply-for-legal-aid/templates/_envs.tpl#L72-L73) and [get paid CCCD](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/13a8140c35ba841d9172241fe0c99d1572d68b34/kubernetes_deploy/jobs/migrate.yaml#L20-L21)

Comment (richrace) [reply helm_deploy/laa-court-data-adaptor/templates/_environment.tpl:18]: The main reason why it looks odd to me, is that we have dev, test, uat, stage and production all in the Cloud Platform. So seeing this hardcoded makes me wonder what it is in those envs?

Comment (richrace) [reply helm_deploy/laa-court-data-adaptor/templates/_environment.tpl:18]: If this is the accepted LAA way of doing things, then that's okay.

Comment (archferns) [reply helm_deploy/laa-court-data-adaptor/templates/_environment.tpl:18]: > The main reason why it looks odd to me, is that we have dev, test, uat, stage and production all in the Cloud Platform. So seeing this hardcoded makes me wonder what it is in those envs?

About the dev, test, uat, stage and prod, the difference is defined via k8s secrets/helm values (eg:  SQS queues and API urls)

Comment (richrace) [reply helm_deploy/laa-court-data-adaptor/templates/_environment.tpl:18]: Yea, typically I'd see the real envs to match those here: https://github.com/ministryofjustice/laa-court-data-adaptor/tree/master/config/environments

This is definitely a different conversation.

**Pull request merged**